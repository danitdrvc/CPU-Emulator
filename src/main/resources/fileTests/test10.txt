MOV QWORD R0, 3
CMP BYTE R0, R1
JGE QWORD label
ADD QWORD R2, R1
label:
HALT