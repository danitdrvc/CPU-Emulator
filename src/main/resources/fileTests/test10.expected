[3, 5, 0, 255]